src/Tmds.Ssh/SshConfig.cs

```diff
@@ -303,6 +311,27 @@ private static void ConfigureFromConfigFile(SshConfig config, string host, strin
                         ThrowUnsupportedKeyword(keyword, remainder);
                         break;
 
+                    case "stricthostkeychecking":
+                        if (!config.HostKeyChecking.HasValue)
+                        {
+                            ReadOnlySpan<char> value = GetKeywordValue(keyword, ref remainder);
+
+                            if (value.Equals("no", StringComparison.OrdinalIgnoreCase) ||
+                                value.Equals("off", StringComparison.OrdinalIgnoreCase))
+                            {
+                                config.HostKeyChecking = StrictHostKeyChecking.No;
+                            }
+                            else if (value.Equals("accept-new", StringComparison.OrdinalIgnoreCase))
+                            {
+                                config.HostKeyChecking = StrictHostKeyChecking.AcceptNew;
+                            }
+                            else
```

Comment:
Currently the parsing is very permissive and I've opted for the "default" case to be some strict value rather than throwing. I want to make this more strict so it behaves more closely to OpenSSH.

Reply:
Fair enough, I just saw some throws for unsupported keywords so thought you wanted to be strict here but I can understand the logic for being restricted for unknown values.

Reply:
I've updated the parsing to be more strict.